src/utilities/scheduler-helper.ts

```diff
@@ -0,0 +1,157 @@
+import axios, { AxiosInstance, AxiosResponse } from 'axios';
+import * as path from 'path';
+
+const DEFAULT_SCHEDULER_URL = 'http://localhost:3003';
+
+const BASE_PATH = '';
+const SET_JOB_ENDPOINT = path.posix.join(BASE_PATH, '/set_job/');
+const LIST_SCHEDULED_JOBS_ENDPOINT = path.posix.join(BASE_PATH, '/list_scheduled_jobs/');
+const LIST_FAILED_JOBS_ENDPOINT = path.posix.join(BASE_PATH, '/list_failed_jobs/');
+const GET_JOB_ENDPOINT = path.posix.join(BASE_PATH, '/get_job/');
+const DELETE_EVENT_ENDPOINT = path.posix.join(BASE_PATH, '/delete_job/');
+
+export interface SchedulerHelperConfigurations {
+    apiKey: string;
+    baseURL?: string;
+};
+
+export enum HttpMethod {
+    GET = 'GET',
+    POST = 'POST',
+}
+
+export interface WebHookScheduleEvent {
+    // NOT IMPLEMENTED ON SCHEDULER SIDE
+    method: HttpMethod;
+    url: string;
+    // NOT IMPLEMENTED ON SCHEDULER SIDE
+    headers?: string;
+    data?: unknown;
```

Comment:
Should you hoist this above the others and have one NOT IMPLEMENTED AFTER comment?